include/veriblock/base58.h

```diff
@@ -0,0 +1,13 @@
+#ifndef _VBCRYPTO
+#define _VBCRYPTO
```

Comment:
did you write this include guard yourself?
if so, please use filename, such as `BASE58_HPP_` or similar

Reply:
Fixed